api/src/main/java/com/thesurvey/api/service/AnsweredQuestionService.java

```diff
@@ -50,8 +51,8 @@ public AnsweredQuestionService(SurveyRepository surveyRepository,
     }
 
     @Transactional
-    public void createAnswer(Authentication authentication,
-        AnsweredQuestionRequestDto answeredQuestionRequestDto) {
+    public void createAnswer(AnsweredQuestionRequestDto answeredQuestionRequestDto) {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
```

Comment:
What is the difference between sending Authentication parameter and getting authentication instance like on this code?

Reply:
I think these two are just the same. Maybe there might be some better way to resolve this.